oss_src/unity/python/sframe/cython/cy_sarray_builder.pxd

```diff
@@ -0,0 +1,43 @@
+'''
```

Comment:
Feel slightly mixed about this. Might prefer this to go via the SDK interface rather than the cython interface. On the other hand the cython interface is probably still going to be somewhat faster.